### DISCUSSION TITLE
Hyperlink link to localhost but not to online website.

### DISCUSSION THREAD

**Discussion by lizhiwei1994** [quarto-dev/quarto-cli#2172] [Q&A] [answered]
Hi everyone.

# Question
Why the hyperlink link to localhost but not to online website.

How can I make the hyperlink link to online website.

# Reproducible code
~~~
---
title: "test hyperlink"
format: revealjs
editor: visual
---

## Quarto

some text^[[A footnote that have a hyperlink](quarto.org)]

[some text that have a hyperlink](quarto.org)
~~~

**Comment by rgaiacs** [comment, marked as answer]
Links to online websites must start with `http://` or `https://`.
